WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModel.swift

```diff
@@ -294,8 +296,9 @@ extension ProductFormViewModel {
         return isSitePublic && formType != .add && productHasLinkToShare
     }
 
+    /// Merchants can promote a product with Blaze if product and site are eligible, and there's no existing Blaze campaign for the product.
     func canPromoteWithBlaze() -> Bool {
```

Comment:
Opinion: Now that we have added an extra condition, what do you think about renaming this into `shouldShowPromoteWithBlaze`? I hope this is only about deciding the promote button's visibility.

Reply:
This is a good suggestion. There's a bit of `canPromoteWithBlaze` name usage both in the viewmodel and in the product form actions factory, and it's a bit late for me today to check everything to make sure `shouldShowPromoteWithBlaze` makes sense everywhere. I'll push it to a separate issue. Thanks!